src/circuit/floor_planner/single_pass.rs

```diff
@@ -226,6 +226,20 @@ impl<'r, 'a, F: Field, CS: Assignment<F> + 'a> RegionLayouter<F>
         Ok((cell, value))
     }
 
+    fn constrain_instance<'v>(
+        &mut self,
+        cell: Cell,
+        instance: Column<Instance>,
+        row: usize,
+    ) -> Result<(), Error> {
+        self.layouter.cs.copy(
+            cell.column,
+            *self.layouter.regions[*cell.region_index] + cell.row_offset,
+            instance.into(),
+            row,
+        )
```

Comment:
```suggestion
        self.layouter.constrain_instance(
            cell,
            instance,
            row,
        )
```
(plus whatever `rustfmt` wants)